doc/classes/PackedByteArray.xml

```diff
@@ -151,6 +151,9 @@
 			<param index="0" name="byte_offset" type="int" />
 			<description>
 				Decodes a 64-bit unsigned integer number from the bytes starting at [param byte_offset]. Fails if the byte count is insufficient. Returns [code]0[/code] if a valid number can't be decoded.
+				[b]Warning:[/b] Since GDScript's [int] is a 64-bit signed integer, the result will be automatically cast to a signed value when ran in GDSCript. Unsigned 64-bit integers can represent numbers in the range [code][0, 2^64 - 1][/code], whereas signed ones can represent numbers in the range [code][-(2^63), 2^63 - 1][/code]. This means:
```

Comment:
```suggestion
				[b]Warning:[/b] Since GDScript's [int] is a 64-bit signed integer, the result will be automatically cast to a signed value when ran in GDScript. Unsigned 64-bit integers can represent numbers in the range [code][0, 2^64 - 1][/code], whereas signed ones can represent numbers in the range [code][-(2^63), 2^63 - 1][/code]. This means:
```